### PR TITLE
[mqtt.generic] Include availabilityTopic in the binding docs

### PR DESCRIPTION
**Pull request by jimtng**
Add availabilityTopics to the documentation for mqtt.generic. This was discussed in https://community.openhab.org/t/mqtt-lwt-things-definition-availability-topic/103065/27

### CODE REVIEW

File: bundles/org.openhab.binding.mqtt.generic/README.md [hunk 1]
```diff
@@ -250,6 +257,15 @@ Thing mqtt:topic:mything "mything" (mqtt:broker:myUnsecureBroker) {
     Type dimmer : blind "Blind" [ stateTopic="blind/state", commandTopic="blind/set", min=0, max=5, step=1 ]
 }
 ```
+
+If the availability status is available, it can be configured to set the Thing status:
+
+```
+Thing mqtt:topic:bedroom1-switch (mqtt:broker:myUnsecureBroker) [ availabilityTopic="tele/bedroom1-switch/LWT", payloadAvailable="Online", payloadNotAvailable="Offline" ] {
+    Channels:
+         Type switch        : power        [ stateTopic="stat/bedroom1-switch/RESULT", transformationPattern="REGEX:(.*POWER.*)∩JSONPATH:$.POWER", commandTopic="cmnd/bedroom1-switch/POWER" ]
```
Comment (lolodomo) [inline bundles/org.openhab.binding.mqtt.generic/README.md:266]: Just for confirmation: `stat/` ?

**Review by lolodomo — COMMENTED** [review]
Just a question for confirmation

File: bundles/org.openhab.binding.mqtt.generic/README.md (hunk 1 above)
Comment (jimtng) [reply bundles/org.openhab.binding.mqtt.generic/README.md:266]: Are you referring to `stateTopic="stat/bedroom1-switch/RESULT"`? This is correct. This was taken straight out of my working things file for a Tasmota device. The topic is `stat/` not `state/` or as you might think. It seems to like 4 character topic prefixes: `stat`, `tele`, and `cmnd`

Comment (lolodomo) [reply bundles/org.openhab.binding.mqtt.generic/README.md:266]: Ok

**Review by lolodomo — APPROVED** [review]
LGTM

**Pull request merged**